Support/Scripts/run-lldb-tests.sh

```diff
@@ -19,11 +19,11 @@ cherry_pick_patches() {
   # Disabled and enabled tests
   local testingPath="$top/Support/Testing"
 
-  if [[ "$platform_name" = "android" ]]; then
+  #if [[ "$platform_name" = "android" ]]; then
```

Comment:
You should delete this block if it's not needed anymore instead of just commenting it.

Reply:
Not a permanent change. Just need to fix the patches given that the files changed in lldb such that the patch no longer works.